### PR TITLE
Logging exports

### PR DESCRIPTION
**Pull request by kent1D**
## country.csv

Better readability of this file:

* first column is ISO
* ordered by ISO code
* use ISO code for sovereign and not "french shorthand" (UK for United Kingdom and not RU for Royaume-Uni, avoid confusion with Russia)

Better name for FYROM / ARYM

## make.py

More logs
Export files in an exports/ folder
Use overpass api over https
Make file names of downloaded datas more readable and usable

### CODE REVIEW

File: make.py [hunk 1]
```diff
@@ -96,9 +96,11 @@
 
 
 @cli
-async def process(itl_path: Path=Path('boundary.json'),
-                  disputed_path: Path=Path('disputed.json'),
+async def process(itl_path: Path=Path('exports/boundary.json'),
+                  disputed_path: Path=Path('exports/disputed.json'),
                   database='mae'):
+    exports = Path('exports')
```
Comment (yohanboniface) [inline make.py]: You can do something like:

    `itl_path.parent.mkdir(…)`

File: make.py [hunk 2]
```diff
@@ -9,16 +9,16 @@
...
+OVERPASS = 'https://overpass-api.de/api/interpreter'
 
 async def get_relation(conn, **tags):
     if 'iso' in tags:
         tags['ISO3166-1:alpha2'] = tags.pop('iso')
     tags = "".join(f'["{k}"="{v}"]' for k, v in tags.items())
     path = Path('tmp/boundary')
     path.mkdir(parents=True, exist_ok=True)
-    path = path / tags.replace('/', '_')
+    file = tags.replace('/', '_').replace('][', '_').replace('"','').replace(':','_').replace('[', '').replace(']', '')+'.json'
```
Comment (yohanboniface) [inline make.py]: `file` is a python keyword, so the convention is to name it `file_` not to override it.

**Review by yohanboniface — APPROVED** [review]
r+wc :)

File: make.py (hunk 1 above)
Comment (kent1D) [reply make.py]: Ok sure, it's replaced

File: make.py (hunk 2 above)
Comment (kent1D) [reply make.py]: Oh sorry I forgot it, I changed it to `file_`

**Comment by kent1D** [comment]
I was just doing it after my comment :)

**Pull request merged**